RelVal/ReleaseValidation.C

```diff
@@ -345,45 +346,66 @@ void WriteHisto2D(TObject *obj) {
 }
 
 void WriteTEfficiency(TObject *obj) { // should I further develop that?
+
   TEfficiency *hEff = static_cast<TEfficiency *>(obj);
+
+  // separate numerator and denominator of the efficiency
   TH1 *hEffNomin  = (TH1*)hEff->GetPassedHistogram(); // eff nominator
   TH1 *hEffDenom = (TH1*)hEff->GetTotalHistogram();   // eff denominator
+
+  // recreate the efficiency dividing numerator for denominator:
+  TH1* heff = dynamic_cast<TH1*>(hEffNomin->Clone("heff"));
```

Comment:
I would not use a dynamic_cast, just `TH1* heff = (TH1*)(hEffNomin->Clone("heff"));`